Kickstarter-iOS/Views/Controllers/ManagePledgePaymentMethodView.swift

```diff
@@ -110,19 +110,19 @@ final class ManagePledgePaymentMethodView: UIView {
 
 // MARK: - Styles
 
-private let cardExpirationDateLabelStyle: LabelStyle = { label in
+private let expirationDateLabelStyle: LabelStyle = { label in
```

Comment:
`expirationDateLabel` & `lastFourDigitsLabel` styles make it simpler when they don't inherit from `checkoutTitleLabelStyle` because these styles were overriding more properties than they were setting...so we're setting these explicitly here per style.